### PR TITLE
Follow up on FTS PR

### PR DESCRIPTION
**Pull request by enoch85**
cc @szaimen @Ark74 Please test current setup and fix all bugs in this PR.

Fix https://github.com/nextcloud/vm/issues/2210

- [x] https://github.com/nextcloud/vm/pull/2197#issuecomment-1024453747
- [x] https://github.com/nextcloud/vm/pull/2197#discussion_r795539391
- [x] This
image

### CODE REVIEW

**Comment by enoch85** [comment]
cc @daita

File: nextcloud_update.sh [hunk 1]
```diff
@@ -586,6 +586,7 @@ $DOCKER_RUN_OUTPUT"
     docker_update_specific 'onlyoffice' 'OnlyOffice'
     # Full Text Search
     docker_update_specific 'fts_esror' 'Full Text Search'
+    docker_update_specific 'fts_os-node' 'Full Text Search'
```
Comment (szaimen) [inline nextcloud_update.sh:589]: @Ark74 I guess this will not work for a docker-compose container, does it?

Comment (enoch85) [reply nextcloud_update.sh:589]: Nope, but I add it there anyway. Will have a look later. If you want you can check as well.

**Comment by Ark74** [comment]
@enoch85 did you moved the certs to OPNSDIR?

The change from statics to apps broke it last time I saw.

**Comment by Ark74** [comment]
@ArtificialOwl the error is because of setup adjustments on the script, the app index correctly.

**Comment by Ark74** [comment]
@enoch85 
Added PR #2213

**Pull request merged**